modules/ossm-kiali-ossmc-install-cli.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-kiali-ossmc.adoc
+
+:_content-type: PROCEDURE
+[id="ossm-kiali-ossmc-install-web-cli_{context}"]
+= Installing OpenShift Service Mesh Console Plugin using CLI
+
+.Prerequesites
+
+The OpenShift Service Mesh Console plugin requires OpenShift v4.10+ with one of:
+
+* Istio v1.14+ with Kiali v1.57+
+* OpenShift Service Mesh (OSSM) v2.3+
+* Kiali Server
```

Comment:
This has changed and will not be correct when we release.  See: https://github.com/kiali/openshift-servicemesh-plugin/blob/main/docs/install-guide.md

We haven't actually documented the latest versions in that updated install guide. But it will probably be something like: Istio v1.19+ (or OSSM v2.5) with Kiali v1.73+.